code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -33,7 +33,7 @@
 	icon_state = "trashbag"
 	item_state = "trashbag"
 
-	w_class = 4
+	w_class = 3

```

Comment:
Where are we supposed to store the Janibelt, then? Putting it inside our bag makes having a Janibelt kinda redundant.